src/python/pants/backend/python/rules/pex.py

```diff
@@ -18,8 +18,6 @@
     TypeVar,
 )
 
-from typing_extensions import Protocol
```

Comment:
Any idea what's up with this? This is a bad change. `typing_extensions` is third party.